index.html

```diff
@@ -301,6 +301,55 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+A series of at least the following registries that support the normative
+deliverables of the Working Group will be published and maintained:
```

Comment:
Incorporating comments from @iherman and @Sakurann --
```suggestion
A series of registries that support the normative deliverables of the Working
Group, which are expected to include those listed below, and may include such
others as the Working Group finds appropriate in the course of producing its
normative deliverables, will be published and maintained under the
[W3C Registry Track](https://www.w3.org/2021/Process-20211102/#registries):
```

Reply:
Or:

```suggestion
The Working Group will deliver a set of registries on the [W3C Registry
Track](https://www.w3.org/2021/Process-20211102/#registries) to
support extension points in the above normative deliverables. The
Working Group expects to produce the following registries, but it will
adjust this list as needed to support the normative deliverables:
```

@iherman, are there any other charters that mention registries that this one could imitate? Or is this group going first and setting the precedent? :)

Reply:
Implemented in c47ffd50ad6224d41e6bc7deabb627aef7a78db2.

Reply:
FYI, I found another charter that's doing registries: [https://www.w3.org/2022/02/proposed-timed-text-wg-charter.html says](https://www.w3.org/2022/02/proposed-timed-text-wg-charter.html#ig-other-deliverables):

> The Working Group MAY create [registry definitions](https://www.w3.org/2021/Process-20211102/#registry-definition) and [registry tables](https://www.w3.org/2021/Process-20211102/#registry-table) and MAY create such [registries](https://www.w3.org/2021/Process-20211102/#registry) to replace or augment existing registries created prior to the formalisation of the Registry track.